pkg/providers/docker/reconciler/reconciler.go

```diff
@@ -61,8 +63,29 @@ func (r *Reconciler) ReconcileCNI(ctx context.Context, log logr.Logger, clusterS
 
 // ReconcileWorkerNodes validates the cluster definition and reconciles the worker nodes
 // to the desired state.
-func (r *Reconciler) ReconcileWorkerNodes(ctx context.Context, log logr.Logger, cluster *anywherev1.Cluster) (controller.Result, error) {
-	return controller.Result{}, nil
+func (r *Reconciler) ReconcileWorkerNodes(ctx context.Context, log logr.Logger, c *anywherev1.Cluster) (controller.Result, error) {
+	log = log.WithValues("provider", "docker", "reconcile type", "workers")
+	clusterSpec, err := cluster.BuildSpec(ctx, clientutil.NewKubeClient(r.client), c)
+	if err != nil {
+		return controller.Result{}, fmt.Errorf("Failed to construct cluster Spec: %v", err)
```

Comment:
```suggestion
		return controller.Result{}, fmt.Errorf("failed to construct cluster Spec: %v", err)
```
The errors can usually get added on to previous error statements.